### PR TITLE
Add Logs to the UI

### PR DESCRIPTION
**Pull request by shaardie**
This patchs adds a new table to the UI to display the logs, if configured.

### CODE REVIEW

File: ui/func.js [hunk 1]
```diff
@@ -149,6 +150,17 @@ var update_data = function () {
...
         });
+
+    axios
+        .get('/api/logs')
+        .then(response => {
+            data.logs = response.data.data[0].attributes.lines;
+        })
+        .catch(error => {
+            if (error.response.status != 404) {
+                console.log(error);
```
Comment (lkiesow) [inline ui/func.js]: This probably shouldn't be a `console.log` but either

```js
throw error;
```

or

```js
console.error(error);
```

Not sure what's better. I would probably use the re-throw, but that's not really based on anything but a vague feeling that that's the right way

**Comment by lkiesow** [comment]
I think the table looks a bit weird and might be somewhat confusing when dealing with adjecent lines containing related information. What about making  this a text block with scrolling overflow instead:

Table:
Screenshot from 2020-07-10 23-06-11

Text block:
Screenshot from 2020-07-10 23-14-33

This seems more natural for logs.

The change would be something like:

```diff
diff --git a/etc/pyca.conf b/etc/pyca.conf
index 7d66288..1ff252c 100644
--- a/etc/pyca.conf
+++ b/etc/pyca.conf
@@ -209,7 +209,7 @@ password         = 'CHANGE_ME'
 # Defining no command will effectively disable the endpoint.
 # Type: string
 # Default: no command
-#log_command = 'journalctl -n 50 -u "pyca*"'
+log_command = 'journalctl -n 50 -u "pyca*"'
 
 
 [logging]
diff --git a/ui/index.html b/ui/index.html
index db67187..7ce270c 100644
--- a/ui/index.html
+++ b/ui/index.html
@@ -103,13 +103,13 @@
 
             <section v-if="logs.length > 0">
                 <h2>Logs</h2>
-                <table>
-                    <tbody>
-                        <template v-for="item in logs"  v-bind:metric="item">
-                            <tr><td>{{ item }}</td></tr>
+                <div class=logs>
+                    <pre>
+                        <template v-for="item in logs" v-bind:metric="item">
+                            {{ item }}<br />
                         </template>
-                    </tbody>
-                </table>
+                    </pre>
+                </div>
             </section>
         </main>
     </div>
diff --git a/ui/style.css b/ui/style.css
index 534226a..e16182e 100644
--- a/ui/style.css
+++ b/ui/style.css
@@ -126,3 +126,14 @@ div.event_status {
 div.event_status span {
     color: red;
 }
+
+div.logs {
+    width: 100%;
+    overflow-x: scroll;
+    border: 1px solid silver;
+}
+
+div.logs pre {
+    white-space: nowrap;
+    padding: 10px;
+    font-family: sans;
+}

```

File: ui/func.js (hunk 1 above)
Comment (lkiesow) [reply ui/func.js]:
```suggestion
                console.error(error);
```

**Comment by shaardie** [comment]
We can do that. Updated the Pull Request.

**Pull request merged**